### ISSUE TITLE
oh-my-zsh aliases no longer autocomplete

### ISSUE THREAD

**Issue by miketheman** [withfig/fig#1731]
### Sanity checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and tested again (tell us if that fixed it)

### Issue Details

```shellsession
❯ alias | grep gl=
gl='git pull'
```

Typing `git pull` presents the autocompletion popup. Typing `gl` presents nothing.

This is pretty uniform for all of my aliases, and it used to work.

### Environment

```yaml
### Details:
|OS|Fig|Shell|
|-|-|-|
|macOS 12.5.1 (21G83)|1.0.60|/bin/zsh|
fig diagnostic

# Fig Diagnostics
## Fig details:
  - Fig version: Version 1.0.60 (B507) [U.S.]
  - Bundle path: /Applications/Fig.app
  - Autocomplete: true
  - Settings.json: true
  - Accessibility: true
  - Number of specs: 0
  - Symlinked dotfiles: true
  - Only insert on tab: false
  - Keybindings path: 
  - Installation Script: true
  - PseudoTerminal Path: /usr/local/opt/ruby/bin:/usr/local/sbin:/opt/homebrew/Cellar/pyenv-virtualenv/1.1.5/shims:/Users/mike.fiedler/.pyenv/shims:/Users/mike.fiedler/.nvm/versions/node/v14.16.1/bin:/Users/mike.fiedler/.pyenv/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Applications/Little Snitch.app/Contents/Components:/Library/Apple/usr/bin:/Users/mike.fiedler/.fig/bin:/Users/mike.fiedler/.local/bin
  - SecureKeyboardInput: false
  - SecureKeyboardProcess: <none>
## Hardware Info:
  - Model Name: MacBook Pro
  - Model Identifier: MacBookPro18,1
  - Chip: Apple M1 Pro
  - Cores: 10 (8 performance and 2 efficiency)
  - Memory: 16 GB
## OS Info:
  - macOS 12.5.1 (21G83)
## Environment:
  - User Shell: /bin/zsh
  - Current Directory: /Users/mike.fiedler
  - CLI Installed: true
  - Executable Location: /opt/homebrew/bin/fig
  - Current Window ID: 4638/% (com.googlecode.iterm2)
  - Active Process: zsh (61162) - not a tty
  - Terminal: iterm
  - Installed via Brew: true
  - Environment Variables:
    - FIG_INTEGRATION_VERSION=8
    - FIG_TERM=1
    - FIG_TERM_VERSION=5.1.0
    - PATH=/usr/local/opt/ruby/bin:/usr/local/sbin:/opt/homebrew/Cellar/pyenv-virtualenv/1.1.5/shims:/Users/mike.fiedler/.pyenv/shims:/Users/mike.fiedler/.nvm/versions/node/v14.16.1/bin:/Users/mike.fiedler/.pyenv/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Applications/Little Snitch.app/Contents/Components:/Library/Apple/usr/bin:/Users/mike.fiedler/.fig/bin:/Users/mike.fiedler/.local/bin
    - TERM=xterm-256color
    - TERM_SESSION_ID=w0t0p0:B9ED267C-2055-4D87-8969-9C53A4AB76D2
    - FIG_PID=61162
## Integrations:
  - SSH: false
  - TMUX: false
  - iTerm: installed!
  - Hyper: application is not present.
  - Visual Studio Code: installed!
  - Docker: true
```

**Comment by SeparateRecords** [comment]
Just to quickly check, are you on the beta version? Wouldn't have expected this to randomly stop working.

Also if you want to set up a time to debug this 1:1, https://calendly.com/robert-fig/support

**Comment by miketheman** [comment]
Per the diagnostic above, I'm on Version 1.0.60 (B507) [U.S.]

**Comment by miketheman** [comment]
Seems resolved with Version 1.0.61 (B518) [U.S.]
